v2/internal/frontend/desktop/windows/frontend.go

```diff
@@ -91,12 +91,18 @@ func NewFrontend(ctx context.Context, appoptions *options.App, myLogger *logger.
 		return result
 	}
 
-	bindingsJSON, err := appBindings.ToJSON()
-	if err != nil {
-		log.Fatal(err)
+	var bindings string
```

Comment:
I'm not up with Go var handling; what happens if `bindings` never get set a value, just a type, then line 105 runs?

Reply:
yes. But that's fine